views/lexicon/forms.html.twig

```diff
@@ -53,10 +53,10 @@
             "icon": "check"
         },
         "3": {
-            "label": "Docs",
+            "label": "Tables",
```

Comment:
Change to single quotes (assuming that's the preferred type in twig based on the other files)